protobufs/livekit_phone_number.proto

```diff
@@ -0,0 +1,129 @@
+message ListPhoneNumberInventoryRequest {
+  string country_code = 1;        // Optional: Filter by country code (e.g., "US", "CA")
+  string area_code = 2;           // Optional: Filter by area code (e.g., "415")
+  int32 limit = 3;                // Optional: Maximum number of results (default: 50)
+  TokenPagination page_token = 4;          // Optional: Token for pagination (empty for first page)
+}
+
+// ListPhoneNumberInventoryResponse - Response containing available phone numbers
+message ListPhoneNumberInventoryResponse {
+  repeated PhoneNumberInventoryItem items = 1;  // List of available phone numbers
+  TokenPagination next_page_token = 2;                   // Token for next page (empty if no more pages)
+}
+
+// PurchasePhoneNumberRequest - Request to purchase phone numbers
+message PurchasePhoneNumberRequest {
+  repeated string phone_numbers = 1;  // Phone numbers to purchase (e.g., ["+1234567890", "+1234567891"])
+}
+
+// PurchasePhoneNumberResponse - Response containing the purchased phone numbers
+message PurchasePhoneNumberResponse {
+  repeated PurchasedPhoneNumber phone_numbers = 1;  // Details of the purchased phone numbers
+}
+
+// ListPurchasedPhoneNumbersRequest - Request to list purchased phone numbers
+message ListPurchasedPhoneNumbersRequest {
+  int32 limit = 1;                // Optional: Maximum number of results (default: 50)
+  TokenPagination page_token = 2;          // Optional: Token for pagination (empty for first page)
+}
+
+// ListPurchasedPhoneNumbersResponse - Response containing purchased phone numbers
+message ListPurchasedPhoneNumbersResponse {
+  repeated PurchasedPhoneNumber items = 1;  // List of purchased phone numbers
+  TokenPagination next_page_token = 2;               // Token for next page (empty if no more pages)
+  int32 total_count = 3;                    // Total number of purchased phone numbers
+}
+
+// ReleasePhoneNumberRequest - Request to release purchased phone numbers
+message ReleasePhoneNumberRequest {
+  repeated string phone_numbers = 1;  // Phone numbers to release (e.g., ["+1234567890", "+1234567891"])
+}
+
+// GlobalPhoneNumber represents a phone number with standardized format
+message GlobalPhoneNumber {
+  string id = 1;                  // unique identifier
+  string e164_format = 2;         // e.g., "+14155552671"
+  string country_code = 3;        // e.g., "US"
+  string area_code = 4;           // e.g., "415"
+  string number_type  = 5;        // e.g., local, toll-free, national, mobile
+  string locality = 6;            // City/locality (e.g., "San Francisco")
+  string region = 7;              // State/region (e.g., "CA")
+  double spam_score = 8;          // can be used later for fraud detection
+  google.protobuf.Timestamp created_at = 9;           // timestamp when created
+  google.protobuf.Timestamp updated_at = 10;          // timestamp when updated
+}
+
+// TelephonyCost represents the pricing structure for a specific telephony service
+message TelephonyCost {
```

Comment:
I don't think this should be exposed. our other APIs don't expose cost/pricing information currently.

when we do, it should not be dependent on a particular resource. but rather types of resources.